File: python_ruby/Calc/1.rb
class Calc
  def initialize(v1,v2)
    @v1 = v1
    @v2 = v2
  end

  def add()
    return @v1 + @v2
  end

  def subtract()
    return @v1 - @v2
  end
end

c1 = Calc.new(10,10)
p c1.add()
p c1.subtract()

c2 = Calc.new(10,20)
p c2.add()
p c2.subtract()

File: python_ruby/Calc/1.py
class Calc(object):
    def __init__(self, v1, v2):
        self.v1 = v1
        self.v2 = v2

    def add(self):
        return self.v1 + self.v2

    def subtract(self):
        return self.v1 - self.v2

c1 = Calc(10,10)
print(c1.add())
print(c1.subtract())

c2 = Calc(20,30)
print(c2.add())
print(c2.subtract())
